leptonai/api/queue.py

```diff
@@ -0,0 +1,44 @@
+# Because most of the apis in queue do not return json, we will not use json_or_error here.
+from typing import Union
+
+from .connection import Connection
+
+
+def list_queue(conn: Connection):
+    """
+    List queues in the current workspace.
+    """
+    response = conn.get("/queue")
+    return response
+
+
+def create_queue(conn: Connection, name: str):
+    """
+    Create a queue in the current workspace.
+    """
+    response = conn.post("/queue", json={"name": name})
+    return response
+
+
+def delete_queue(conn: Connection, name: str):
+    """
+    Delete a queue from the current workspace.
+    """
+    response = conn.delete(f"/queue/{name}")
+    return response
+
+
+def receive(conn: Connection, name: str):
+    """
+    Receives a message from a queue.
+    """
+    response = conn.get(f"/queue/{name}/messages")
+    return response
+
+
+def send(conn: Connection, name: str, message: Union[str, bytes]):
```

Comment:
I think message can only be of type `str`? Posting bytes in json will raise error IIRC

Reply:
yeah... will fox.